### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Saibamen**
Powered by markdownlint

### CODE REVIEW

**Comment by Saibamen** [comment]
@HBelusca New commit

File: README.md [hunk 1]
```diff
@@ -44,14 +44,14 @@ The ReactOS project, although currently focused on Windows Server 2003 compatibi
...
 
-## Building 
+## Building
 
 [![appveyor.badge]][appveyor.link] [![travis.badge]][travis.link] [![rosbewin.badge]][rosbewin.link] [![rosbeunix.badge]][rosbeunix.link] [![coverity.badge]][coverity.link]
 
-To build the system it is strongly advised to use the _ReactOS Build Environment (RosBE)._ 
-Up-to-date versions for Windows and for Unix/GNU-Linux are available from our download page at: http://www.reactos.org/wiki/Build_Environment.
+To build the system it is strongly advised to use the _ReactOS Build Environment (RosBE)._
+Up-to-date versions for Windows and for Unix/GNU-Linux are available from our download page at: [http://www.reactos.org/wiki/Build_Environment](Build Environment).
```
Comment (HBelusca) [inline README.md]: Checking the other examples, you can see it's the other way around: `[Human-readable description](http://www.url.link)`.

File: README.md [hunk 2]
```diff
@@ -44,14 +44,14 @@ The ReactOS project, although currently focused on Windows Server 2003 compatibi
...
 
-Alternatively one can use Microsoft Visual C++ (MSVC) version 2010+. Building with MSVC is covered here: https://www.reactos.org/wiki/Building_with_MSVC.
+Alternatively one can use Microsoft Visual C++ (MSVC) version 2010+. Building with MSVC is covered here: [https://www.reactos.org/wiki/CMake#Visual_Studio_or_Microsoft_Visual_C.2B.2B](Visual Studio or Microsoft Visual C++).
```
Comment (HBelusca) [inline README.md]: See comment above.

File: README.md [hunk 3]
```diff
@@ -103,30 +103,30 @@ It is not another wrapper built on Linux, like WINE. It does not attempt or plan
 
 ReactOS is also not "yet another OS". It does not attempt to be a third player like any other alternative OS out there. People are not meant to uninstall Linux and use ReactOS instead; ReactOS is a replacement for Windows users who want a Windows replacement that behaves just like Windows.
 
-More information is available at: https://www.reactos.org.
+More information is available at: [https://www.reactos.org](reactos.org).
```
Comment (HBelusca) [inline README.md]: See comment above.

File: README.md (hunk 2 above)
Comment (HBelusca) [reply README.md]: And +1 for having found the working URL.

File: README.md (hunk 1 above)
Comment (HBelusca) [reply README.md]: Maybe also enclose the human-readable description if it stands on itself (as it is the case here: "Build Environment")

File: README.md [hunk 4]
```diff
@@ -82,7 +82,7 @@ See ["Installing ReactOS"](https://www.reactos.org/wiki/Installing_ReactOS) Wiki
 
 ## Testing
 
-If you discover a bug in ReactOS search on JIRA first - it might be reported already. If not report the bug providing logs and as many information as possible. 
+If you discover a bug in ReactOS search on JIRA first - it might be reported already. If not report the bug providing logs and as many information as possible.
```
Comment (Doug-Lyons) [inline README.md]: The statement "..providing logs and as many information as possible." might be improved by changing it to "... providing logs and as much information as possible."

**Pull request merged**